manifests/job.pp

```diff
@@ -121,7 +121,7 @@
 # Copyright 2014 EvenUp.
 #
 define curator::job (
-  $path                  = '/usr/bin/curator',
+  $command               = '/usr/bin/curator',
```

Comment:
This rename makes this a breaking change.  I agree $path isn't as descriptive, is there a need to rename it at this time?  I'd prefer to not have breaking changes unless there's a need to.